mlflow/pyfunc/model.py

```diff
@@ -400,6 +400,14 @@ def _save_model_with_class_artifacts_params(
         mlflow_model.model_size_bytes = size
     mlflow_model.save(os.path.join(path, MLMODEL_FILE_NAME))
 
+    saved_code_subpath = _validate_infer_and_copy_code_paths(
+        code_paths, path, infer_code_paths, FLAVOR_NAME,
+    )
+    mlflow_model.code = saved_code_subpath
```

Comment:
Does `Model` class has this `code` attribute?

Reply:
Yes. the `MLmodel` yaml file has a field of `code`. So we need to set `code` attribute.

Reply:
```
% grep code mlflow/models/model.py
MODEL_CODE_PATH = "model_code_path"
        .. code-block:: python
        .. code-block:: python
        .. code-block:: python
        .. code-block:: python
    .. code-block:: python
    When logging model as code, this function can be used to set the model object
```

Looks like the Model class doesn't have a `code` attribute.

Reply:
You are right, 
we should use `mlflow_model.flavors[FLAVOR_NAME][CODE] = code_dir_subpath` instead of `mlflow_model.code = saved_code_subpath`